prod/westeurope/internal/api/apim/api_management_api_cgn_merchant/terragrunt.hcl

```diff
@@ -0,0 +1,32 @@
+dependency "api_management" {
+  config_path = "../api_management"
+}
+
+# Internal
+dependency "resource_group" {
+  config_path = "../../../resource_group"
+}
+
+# Include all settings from the root terragrunt.hcl file
+include {
+  path = find_in_parent_folders()
+}
+
+terraform {
+  source = "git::git@github.com:pagopa/io-infrastructure-modules-new.git//azurerm_api_management_api?ref=v2.1.35"
+}
+
+inputs = {
+  name                  = "io-cgn-merchant-api"
+  resource_group_name   = dependency.resource_group.outputs.resource_name
+  api_management_name   = dependency.api_management.outputs.name
+  revision              = "1"
+  display_name          = "IO CGN MERCHANT API"
+  description           = "CGN MERCHANT API for IO platform."
+  host                  = "api.io.italia.it"
+  path                  = "cgnmerchant"
```

Comment:
@alessandrodossi can you please double check this?

Reply:
Hello @uolter, I think I am the wrong guy :)

Reply:
> Hello @uolter, I think I am the wrong guy :)

yep .... sorry.